### PR TITLE
combine F.p.toString and E.p.stack censorship proposals

### LINKED ISSUE #8: Interaction with stacks?
How does a stack frame from within a censored function show up in a stack trace?

How does a stack frame from within a non-censored function (but that is "within" a censored function in the call stack) show up in a stack trace?'

Not sure what the answer should be, but these questions definitely need answering prior to stage 3.

(https://github.com/tc39/proposal-error-stacks)

cc @erights @codehag

### PR DESCRIPTION
**Pull request by michaelficarra**
This PR is a WIP right now. I'm updating section by section as I get time to work on it. I've done the introduction and problem statement so far. I'll add a comment when it's ready for a full review, but feel free to start reviewing now.

Fixes #8.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,3 +142,5 @@ console.assert(foo.toString.includes("...")); // oops
 This basically makes the censorship toothless, so that you do not gain the encapsulation or memory usage benefits.
 
 You could try to patch around it by saying that only setting it to `true` works, and setting it to `false` or deleting the property does nothing. But that's just strange. We already have a mechanism for changing the state of an object in a non-reversible way: call a method on it. Thus, the `foo.censor()` proposal above (which has its own problems).
+
+Additionally, we want the author of the function to be the one in control of its censorship, not anyone with a reference to that function. This was discussed with respect to an API for `Error.prototype.stack` censorship in the January 2019 TC39 meeting.
```
Comment (domenic) [inline README.md]: I think this works better in the "rejected alternatives" section.

Comment (michaelficarra) [reply README.md]: Agreed.

**Comment by michaelficarra** [comment]
Okay @domenic @ljharb @phoddie this should be ready for review now.

**Comment by domenic** [comment]
I've pushed minor review tweaks. I'll also work on updating the spec to at least use the new name.

I'd like to merge this ASAP so I can work on the other outstanding issues in the repository, but I want to make sure you're OK with the changes I made before doing so. Let me know!

**Comment by michaelficarra** [comment]
LGTM

**Comment by domenic** [comment]
@michaelficarra do you want to be a co-champion?

**Comment by michaelficarra** [comment]
@domenic Yes, that seems appropriate.

**Pull request merged**